docs/howto/openstack/neutron/del-network.md

```diff
@@ -0,0 +1,553 @@
+    | e0c4ce17-2722-4777-8140-d6c87479e190 | network-kna1 | 421d8fd2-dd7f-4f7c-9a51-42ef4a866dd9 |
+    +--------------------------------------+--------------+--------------------------------------+
+    ```
+    
+## Networks with no subnet and no router
+
+You may directly, without the slightest preparation, delete networks 
+like these. For our demonstration, we created a network named `mayo`, 
+with no subnet and no router in front of it.
+
+=== "{{gui}}"
+    While viewing all available networks, click the red three-dot icon at 
+    the right-hand side of the `mayo` row and select _Delete Network_. You 
+    will have to confirm the action, and the network will be gone as soon 
+    as you do.
+    
+    ![Delete Mayo](assets/del-net/shot-17.png)
+=== "OpenStack CLI"
+    Once more, take a look at all remaining networks:
+        
+    ```bash
+    openstack network list --internal
+    ```
+    ```plain
+    +--------------------------------------+--------------+--------------------------------------+
+    | ID                                   | Name         | Subnets                              |
+    +--------------------------------------+--------------+--------------------------------------+
+    | 1f94d315-7ca1-4d44-acc1-09c6c650df74 | mayo         |                                      |
+    | e0c4ce17-2722-4777-8140-d6c87479e190 | network-kna1 | 421d8fd2-dd7f-4f7c-9a51-42ef4a866dd9 |
+    +--------------------------------------+--------------+--------------------------------------+
+    ```
+        
+    Since `mayo` has no subnet, issue a single command to delete it:
+        
+    ```bash
+    openstack network delete mayo
+    ```
+        
+    And, yes, it is still a good idea to check yourself:
+        
+    ```bash
+    openstack network list --internal
+    ```
+    ```plain
+    +--------------------------------------+--------------+--------------------------------------+
+    | ID                                   | Name         | Subnets                              |
+    +--------------------------------------+--------------+--------------------------------------+
+    | e0c4ce17-2722-4777-8140-d6c87479e190 | network-kna1 | 421d8fd2-dd7f-4f7c-9a51-42ef4a866dd9 |
+    +--------------------------------------+--------------+--------------------------------------+
+    ```
+
+## Recap: Of networks and towns
+
+Depending on the features of a Neutron network, deleting it may require 
+some preparation work. For the sake of this demonstration, we created 
+three different networks with different characteristics; `carmacks`, 
+`teslin`, and `mayo`. Then, either from the {{gui}} or with the help of 
+OpenStack CLI, we showed how we discover any component dependencies and 
+how we work towards deletion. Eventually, all three test networks were 
+gone. We should point out, though, that all three namesake towns in 
+Yukon are still there.
```

Comment:
You can remove the last sentence. :)

Reply:
Yes, I could change the title of the section, and I could delete that last sentence. But then, why not name the test networks as `network1`, `network2`, and `network3` also? That wouldn't hurt in the least, for sure. On the other hand, I believe that picking non-trivial names and then, at the end of the HOWTO, when no one expects it, adding a reference like the above maybe helps some readers realize that we do enjoy what we're doing. I am not saying that names/references like this are necessary, mind you. All I'm saying is that sometimes it doesn't hurt to show that we do enjoy preparing those guides. This is how I feel, and maybe 100% wrong here. But what do you think?

Reply:
Yeah on second thought I conclude you're right. Let's merge this as per the state of your latest update; I'll just go ahead and rebase this on current `main` before I do.